index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Version Skeleton</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main class="maincontainer">
+        <header>
+            <a href="#" class="logo">Crysty</a>
+            <nav>
```

Comment:
- [ ] [optional] I would suggest putting the `<header>` out of the `<main>.` It's not a good practice to place `<header>` inside the `<main>` because they are intended to be at the same level of hierarchy. The main tag should only be responsible for the main /hero section of the webpage.